### PR TITLE
[Fuzzing] Emit secondary wasm files in ClusterFuzz testcases

### PR DESCRIPTION
**Pull request by kripken**
The two files are then linked and run by `fuzz_shell.js` (we had this functionality
already in order to fuzz `wasm-split`). By adding multiple build and run commands
of both the primary and secondary wasm files, we can end up with multiple
instances of two different wasm files that call between themselves.

To help testing, add a script that extracts the wasm files from the testcase. This
may also be useful in the future for testcase reduction.

### CODE REVIEW

File: scripts/clusterfuzz/extract_wasms.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+
+in_js = sys.argv[1]
+out = sys.argv[2]
+
+with open(in_js) as f:
+    js = f.read()
+
+
+def repl(text):
```
Comment (tlively) [inline scripts/clusterfuzz/extract_wasms.py:53]: I thought this had something to do with read-eval-print loops for a second. How about naming it `replace` instead?

**Review by tlively — COMMENTED** [review]
How do the instances call each other? Just via the table? Can we get more control over that?

**Comment by kripken** [comment]
In addition to calling each other through the table, there is the `call-export` mechanism, an import that is given an export index and calls it, #7067.

File: scripts/clusterfuzz/extract_wasms.py (hunk 1 above)
Comment (tlively) [reply scripts/clusterfuzz/extract_wasms.py:53]: Haha, ok, if that's what the docs use, then SGTM.

**Review by tlively — APPROVED** [review]

**Pull request merged**